src/cookie.css

```diff
@@ -105,3 +105,25 @@ input[type=checkbox]:checked {
 .managed #unknown, .managed button {
     display: none;
 }
+
+.blocked:after {
+    content: "first visit, not trusted yet";
+    color: #666;
+}
+.blocked.visited:after {
+    content: "previously visited, not yet trusted";
+    color: #999;
+}
+.unblocked:after {
+    content: "trusted because visited regularly";
+    color: yellow;
+}
+
+.manual.unblocked:after {
+    content: "trusted manually";
+    color: yellow;
+}
+.manual.blocked:after {
+    content: "blocked manually";
```

Comment:
Recommend -
"Blocked: By Choice"

_I think By Choice is important because they are providing or denying consent_